scripts/valhalla_build_config

```diff
@@ -557,56 +558,67 @@ help_text = {
   }
 }
 
-#used to convert comma separated list of strings into a list
-def comma_separated_list(arg):
-  return arg.split(',')
 
-def parse_bool(arg):
-  if arg.lower() in ('yes', 'true', 't', 'y', '1'):
-    return True
-  elif arg.lower() in ('no', 'false', 'f', 'n', '0'):
-    return False
-  else:
-    raise argparse.ArgumentTypeError('Boolean value expected.')
+def comma_separated_list(arg: str) -> List[str]:
+    """used to convert comma separated list of strings into a list"""
+    return arg.split(',')
```

Comment:
really hard to see what's going on in this diff.. 

- I just moved the comments above the functions into their docstrings
- added the type hints